conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
+                    <ul>
+                        <li>For conflicts that have a clear path to resolution and low severity (for example, a harsh criticism in a meeting or inappropriate language), the focus should be on coming to a shared understanding of what happened and agreeing to a path to resolution (for example, an apology or editing a comment or minutes).</li>
+                    </ul>  
+                </li>
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>Ombudspeople are advisors to participants who can provide guidance, advice, and support during the conflict resolution process. Ombudspeople are meant to be neutral third parties to any incident, and as such are required not to take sides or provide opinions on the indicidents they are involved in.</p>
+
+            <p>An Ombudsperson can advise or assist you with:</p>
+
+            <ul>
+                <li>How the Code of Conduct may apply in your situation</li>
+                <li>How to start the mediation process</li>
+                <li>How to raise a formal complaint</li>
+            </ul>
+
+            <p>TODO: More about the role of the ombudsperson and what they can and cannot do.</p>
+        </section>
+
+        <section id="mediation">
+            <h2>Mediation</h2>
+
+            <p>Mediation is most effective when used earlier in a dispute, before people become fixed in their position, as it provides a positive opportunity to resolve practical problems.</p>
+
+            <p>It empowers people to work together to resolve issues, and can therefore be less stressful and have a less negative impact on individuals. Mediation can also be successful in later stages after an investigation has concluded to help people rebuild relationships.</p>
+                
+            <p>It is usually between two parties, but mediation can also work for groups. Mediation can be used to resolve a range of issues including a breakdown in relationships or ongoing conflicts between group members.</p>
+
+            <p>Where it's appropriate to attempt to resolve an issue with mediation, there are two approaches that can be taken:</p>
+
+            <ul>
+                <li>Informal Mediation: This approach may include solely the parties involved in the incident, or extend to trusted individuals to help facilitate the conversation (for example, chairs or ombudspeople). The process may be more informal in that it takes place over a call or email correspondance, but the goal remains: resolution of conflict.</li>
```

Comment:
If it only involves the parties involved it's direct discussion between the parties, not mediation, so I suggest rewording the above item to:

"Informal Mediation: Any mutually trusted individual may act as an informal mediator, such as a chair or Ombud or even another W3C participant.  The process may take place in person, over a call, through email, or through some other medium, but the goal remains: to reach resolution.